docs/workforce-management-framework/topics/con-PassportAuth-Module.adoc

```diff
@@ -0,0 +1,41 @@
+[]
+= {WFM-RC-NameShort} Passport.js integration module
+Link to
+link:https://github.com/feedhenry-raincatcher/raincatcher-core/tree/master/cloud/passportauth[passportAuth] module.
+
+{WFM-RC-NameShort} authentication and authorisation module using link:http://passportjs.org[PassportJS].
+The Passport.js module is the default security module for {WFM-RC-NameShort}.
+
+Passport is using "pluggable" authentication plugins called link:http://passportjs.org/docs/configure[strategies].
```

Comment:
Passport provides pluggable authentication using link:http://passportjs.org/docs/configure[strategies].

Improved readability.

Reply:
yep thats straight from the passport js docs 
but I agree.